src/components/UserManagement/StatusUserPopup.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
Has this component been used? I did not find a button to trigger this popup.

Reply:
Same here, wrong one, accidentally included.